rust/kcl-lib/src/execution/mod.rs

```diff
@@ -546,14 +546,17 @@ impl ExecutorContext {
                 mem.add(k, v, SourceRange::synthetic())
                     .map_err(KclErrorWithOutputs::no_outputs)?;
             }
+            eprintln!("running {mem}");
```

Comment:
Should this be using the `log()` helper or something?

Reply:
Whoops, no, it just shouldn't be there